frontend/app/components/searchform.jsx

```diff
@@ -1,21 +1,35 @@
 import React from 'react';
-import { Typeahead }  from 'react-bootstrap-typeahead';
+import { AsyncTypeahead }  from 'react-bootstrap-typeahead';
 
 class SearchForm extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-        data: ['these', 'will', 'be' ,'restaurants']
-    };
+        options: []
+        };
   }
 
+  handleSearch(query) {
+    if (!query) {
+      return;
+    }
+
+    fetch('http://localhost:8000/api/v1/businesses/?search=' + query)
```

Comment:
You don't want to ever hard-code your root URL (e.g. `http://localhost:8000`) anywhere in your code. In most cases you can use a URL without a root, like just `/api/v1/businesses/?search=` and it will simply default to the same URL that the file was served from.

If that doesn't work then you need to create an environment-specific settings file for the front-end and load it with webpack. Here is a page describing that technique:

https://m.mattmclaugh.com/application-environment-configuration-with-webpack-238dd0015e42